pyproject.toml

```diff
@@ -2,7 +2,10 @@
 DJANGO_SETTINGS_MODULE = "core.settings"
 env_files = [".env.template"]
 addopts = "--reuse-db"
-filterwarnings = ["error", "ignore::pydantic.warnings.PydanticDeprecatedSince20"]
+filterwarnings = [
+  "ignore::pydantic.warnings.PydanticDeprecatedSince20",
```

Comment:
```suggestion
  # Because of django-ninja package, cf. https://github.com/vitalik/django-ninja/issues/1266
  "ignore::pydantic.warnings.PydanticDeprecatedSince20",
```